package.json

```diff
@@ -29,6 +29,7 @@
     "dotenv": "^16.0.1",
     "graphql": "^15.3.0",
     "json-edit-react": "^0.9.6",
+    "json-to-css": "^0.1.0",
```

Comment:
This package just converts CSS defined in an object to a string.
Eg.
```js
{
  "#user-area": {
    "background-color": "#9c85c4"
  }
}
```
becomes `"#user-area{background-color:#9c85c4}"`, which is what gets injected into the `<head>`

It's a pretty tiny package -- only a handful of lines of code.